hs/t/y/continue-mode.rsh

```diff
@@ -0,0 +1,24 @@
+'reach 0.1';
+
+const f = ((p) => {
+  p.only(() => {
+    const choice = declassify(interact.choice)
+  });
+  p.publish(choice);
+});
+
+export const main = Reach.App(() => {
+  const Alice = Participant('Alice', {
+    choice: UInt,
+  });
+  deploy();
+  f(Alice);
+  var keepGoing = true;
+  invariant(keepGoing);
+  while (keepGoing) {
+    commit();
+    f(Alice)
```

Comment:
We're looking for `keepGoing = f();` where you are in step before `f()` and consensus after `f()` (i.e. inside it is a publish). You don't need to pass Alice... just define the function in this scope right here